vimpager

```diff
@@ -12,16 +12,17 @@ fi
 # FIND REAL PARENT DIRECTORY
 link=$0
 
-while true; do
-    ls0=`ls -l "$link"`
-    new_link=`expr "$ls0" : '.* -> \(.*\)$'`
-
-    [ -z "$new_link" ] && break
-
-    link=$new_link
+while [ -h "$link" ]; do
+    ls0="`ls -l \"$link\"`"
+    new_link="`expr \"$ls0\" : '.* -> \(.*\)$'`"
+    if expr "$new_link" : '/.*' > /dev/null; then
+        link="$new_link"
+    else
+        link="$(dirname $link)/$new_link"
+    fi
 done
 
-project_dir=`dirname "$link"`
+project_dir="`dirname \"$link\"`"
```

Comment:
Yes I would remove them in vimcat as well (I also have some changes like this in #183 but I can just rebase that if yours is merged first).